DP02_16_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.                    Statistic value.
</pre><p>No indentation; see example in <a href="https://developer.lsst.io/python/numpydoc.html#py-docstring-basics" target="_blank">https://developer.lsst.io/python/numpydoc.html#py-docstring-basics</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>adjusted</p>